dance/transforms/normalize.py

```diff
@@ -85,3 +94,379 @@ def __call__(self, data):
         for name, idx in idx_dict.items():
             self.logger.info(f"Scaling {name} (n={len(idx):,})")
             data.data.X[idx] = normalize(data.data.X[idx], mode=self.mode, axis=self.axis, eps=self.eps)
+
+
+class ScTransform2(BaseTransform):
```

Comment:
Can we change this naming? Use `ScTransformR` if it is indeed ScTransform from R instead of `ScTransform2`